modules/n1ql/pages/n1ql-language-reference/transactions.adoc

```diff
@@ -75,25 +75,49 @@ Refer to the documentation for each statement for more information and examples.
 The Query service provides settings and parameters in support of Couchbase transactions.
 Refer to the documentation for each parameter for more information and examples.
 
-* The {txid}[txid] request-level parameter specifies the transaction to which a statement belongs.
+|===
+| Setting{nbsp}/ Parameter | Description
 
-* The {tximplicit}[tximplicit] request-level parameter specifies that a statement is a single transaction.
+| {txid}[txid] request-level parameter
```

Comment:
these variables are interesting: I was curious why not something like `{query-settings}#txid[txid]`?
I suppose having an explicit list of all the link fragment targets in header is useful.

Reply:
I just find it more readable to have the xref attributes as short as possible